packaging_apps_multiinstance.md

```diff
@@ -4,13 +4,13 @@
 Multi-instance is application capacity to be installed several times.
 
 #### Scripts
-When YunoHost installs the application a second time, it gives to the script, at last argument, `${id}__2` with the application `${id}` coming from the manifest. The value `n` in `${id}__n` is incremented each time a new instance of the application is installed.
+When YunoHost installs the application a second time, it pass `$YNH_APP_INSTANCE_NAME` var to the script, set to value `${id}__2` with the application `${id}` coming from the manifest. The value `n` in `${id}__n` is incremented each time a new instance of the application is installed.
```

Comment:
$YNH_APP_INSTANCE_NAME is passed every time, not only the second (or more) time YunoHost installs the app.

Maybe we should talk about $YNH_APP_ID and  $YNH_APP_INSTANCE_NUMBER here